app/views/users/_event_card.html.erb

```diff
@@ -0,0 +1,25 @@
+<% @previous_events.each do |event| %>
+<div class="col-md-4">
+<div class="text-center shadow-lg rounded bg-light py-2">
+<h4 class="text-info font-weight-bold">TITLE</h4>
+<span class="font-weight-normal text-dark"><%= event.title %></span>
+<h4 class="text-info font-weight-bold">description</h4>
+<p><%= event.description %></p>
+<h4 class="text-info font-weight-bold">date</h4>
+<p><%= event.time %></p>
+<div class="w-100 bg-secondary text-light py-2 mt-3 font-weight-bold"><%= link_to 'See Attendees', event %></div>
+<div class="w-100 bg-secondary text-light py-2 mt-1 font-weight-bold">Created by: <%= User.find(event.creator_id).username %></div>
+<% if belong_to_user?(event.creator_id) %>
+<div class="w-100 bg-success text-light py-2 mt-1 font-weight-bold">This is Your Event</div>
+<span><%= link_to 'Edit', edit_event_path(event), class:"text-primary" %> <%= link_to 'Destroy', event, method: :delete, data: { confirm: 'Are you sure?' }, class:"text-danger" %></span>
+<% elsif !is_attending?(event.id) %>
+<%= simple_form_for(@attendance) do |f| %>
+<%= hidden_field_tag(:events_to_attend_id, event.id) %>
+<%= f.button :submit, "Attend", class:"submit-btn" %>
+<% end %>
+<% else %>
+<div class="w-100 bg-warning text-light py-2 mt-1 font-weight-bold">You Attended This Event</div>
+<% end %>
```

Comment:
- [ ] use a helper method to render the if/elseif/else statement in this part of your code there should be no if/elseif/else statements within views if you have several lines you can use a partial to render the markup